src/tools/gophish_complete.py

```diff
@@ -93,15 +93,9 @@ def select_campaigns(campaigns):
     return inputId
 
 
-def complete_campaign(api, workingID):
+def complete_campaign(api, api_key, server, workingID):
     """Complete a campaign."""
-    url = (
-        args["SERVER"]
-        + "/api/campaigns/"
-        + workingID
-        + "/complete?api_key="
-        + args["API_KEY"]
-    )
+    url = f"{server}/api/campaigns/{workingID}/complete?api_key={api_key}"
```

Comment:
🚀 Oh yeah, that looks so much nicer! 👍

Reply:
Agree. My next PR is going to be changing everything to f-strings and the capitalization mentioned above.